extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/runtime/MongoClientRecorder.java

```diff
@@ -266,4 +272,22 @@ private MongoCredential createMongoCredential(MongoClientConfig config) {
                     }
                 }).collect(Collectors.toList());
     }
+
+    private SSLContext getSSLContext(String className) {
+        try {
+            Class clazz = Class.forName(className);
```

Comment:
the Class.forName might be a problem. @stuartwdouglas WDYT? Would that work with your ongoing changes?